x-pack/plugins/session_view/public/components/process_tree/helpers.ts

```diff
@@ -74,12 +73,14 @@ export const buildProcessTree = (
 
   // with this new page of events processed, lets try re-parent any orphans
   orphans?.forEach((process) => {
-    const parentProcess = processMap[process.getDetails().process.parent.entity_id];
+    const parentProcessId = process.getDetails().process.parent?.entity_id;
```

Comment:
@mitodrummer or @opauloh can you take a look at these changes below? Thanks!

Reply:
sure I'm pulling the PR to validate these changes

Reply:
lgtm

Reply:
lgtm ++